{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/sass/base/mixins/_breakpoint.scss

```diff
@@ -0,0 +1,18 @@
+@mixin breakpoint($point) {
+   @if $point == desktop {
```

Comment:
Just include this? https://github.com/Lightmatter/v2fluentcity/blob/develop/fluentcity/static_source/sass/base/_breakpoints.scss